java/j2ee.persistence/src/org/netbeans/modules/j2ee/persistence/wizard/EntityClosure.java

```diff
@@ -425,19 +399,9 @@ public boolean isModelReady() {
     
     void waitModelIsReady(){
         try {
-            Future result = model.runReadActionWhenReady(new MetadataModelAction<EntityMappingsMetadata, Boolean>() {
-
-                @Override
-                public Boolean run(EntityMappingsMetadata metadata) throws Exception {
-                    return true;
-                }
-            });
+            Future result = model.runReadActionWhenReady( (EntityMappingsMetadata metadata) -> true );
             result.get();
-        } catch (InterruptedException ex) {
-            Exceptions.printStackTrace(ex);
-        } catch (ExecutionException ex) {
-            Exceptions.printStackTrace(ex);
-        } catch (MetadataModelException ex) {
+        } catch (InterruptedException | ExecutionException | MetadataModelException ex) {
             Exceptions.printStackTrace(ex);
         } catch (IOException ex) {
```

Comment:
Could be integrated into multi-catch.

Reply:
Done.

Reply:
There is not change here. The IOException still stands alone